generators/app/templates/stories/styles.scss

```diff
@@ -17,3 +17,26 @@
 .input {
   margin-right: 10px;
 }
+
+.checkmark {
+  appearance: none;
+  background-color: #fafafa;
```

Comment:
we could talk for create `scss/colors` and put all de colors of bootstrap.What you think ?

Reply:
I wouldn't add a color that's only in the storybook to the bootstrap, it won't be used in the real app. We could have variables though, or a colors file just for storybook. I'd add that, a constants.scss file to import, inside storybook directory